static/app/views/sharedGroupDetails/sharedGroupHeader.tsx

```diff
@@ -1,19 +1,27 @@
 import styled from '@emotion/styled';
 
 import FeatureBadge from 'sentry/components/badge/featureBadge';
+import {DateTime} from 'sentry/components/dateTime';
 import EventMessage from 'sentry/components/events/eventMessage';
 import ProjectBadge from 'sentry/components/idBadge/projectBadge';
 import ShortId from 'sentry/components/shortId';
+import {Tooltip} from 'sentry/components/tooltip';
 import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
+import type {Event} from 'sentry/types/event';
 import type {Group} from 'sentry/types/group';
 import {IssueCategory} from 'sentry/types/group';
+import getDynamicText from 'sentry/utils/getDynamicText';
+import EventCreatedTooltip from 'sentry/views/issueDetails/eventCreatedTooltip';
 
 type Props = {
   group: Group;
 };
 
 function SharedGroupHeader({group}: Props) {
+  const date = group.latestEvent?.dateReceived ?? group.latestEvent?.dateCreated;
+  const date_obj = new Date(date as string);
+  const event = group.latestEvent as Event;
```

Comment:
Good thinking to cast `group.latestEvent` as an Event to solve the type checking error you got, but this could lead to unexpected consequences down the road. This is because `group.latestEvent` has type `Event | undefined`, which means it _could_ be `undefined`. If it is, then your component will error out since it will attempt to get the `dateCreated` attribute of `undefined` on line 47. 

This is a tricky situation: the `group` object this component receives isn't guaranteed to have a `latestEvent`, but we depend on `latestEvent` to get the last seen timestamp. Perhaps we can somehow modify this component to account for this situation?  Let me know if any of that could use some clarification!